core/controllers/android_e2e_config.py

```diff
@@ -60,22 +85,24 @@ def post(self):
                 raise Exception('The topic is already published.')
             else:
                 raise Exception('The topic exists but is not published.')
-        result = {}
         exp_id = '26'
         user_id = feconf.SYSTEM_COMMITTER_ID
+        # Generating new Structure id for topic, story, skill and question.
```

Comment:
The tense here should be 'Generate' not 'Generating'. Ditto elsewhere in comments.

Reply:
Done